README.md

```diff
@@ -59,9 +59,19 @@ wikibaseintegrator~=0.11.3
 # WikibaseIntegrator / WikidataIntegrator #
 
 WikibaseIntegrator (wbi) is a fork of [WikidataIntegrator](https://github.com/SuLab/WikidataIntegrator) (wdi) whose
-purpose is to focus on compatibility with Wikibase. There have been many improvements which have led to radical changes
-in the code. See the [release notes](https://github.com/LeMyst/WikibaseIntegrator/releases) to find out what has
-changed.
+purpose is to focus on an improved compatibility with Wikibase and adding missing functionalities.
+The main differences between these two libraries are :
+
+* A complete rewrite of the library with an object-oriented architecture allowing for easy interaction, data validation
+  and extended functionality
+* Add support for reading and writing Lexeme, MediaInfo and Property datatypes
+* Add support for [Structured Data on Commons](https://commons.wikimedia.org/wiki/Commons:Structured_data) (SDC)
+* Python 3.7 to 3.11 support, validated with unit tests
+* Type hints implementation for arguments and return
```

Comment:
I see that the library uses `**kwargs: Any` several times ... this very much negates any advantage from parameter type hints, since when you mistype a parameter, nothing will complain. I also see `json_data: Dict[str, Any]` ... I don't think using `Any` there is necessary either ... you should be able to express a `JsonType`.

There are several static type checkers for Python, you could mention that you use `mypy`.